maze-consulting/SFCC_Klaviyo_Maze
# Linter fixes

## Description

This PR resolves the majority of the linter errors flagged in the Initial Check-in of new Klaviyo Cartridge PR ([linked here](https://github.com/klaviyo/SFCC_Klaviyo/pull/25)). It also includes some adjustments to address reviewer feedback.


## Manual Testing Steps

Manual testing was performed during and after these adjustments were incorporated. Tests involved a run through of the six events with the debugger on to check the `dataObj` & `eventData` as breakpoints were hit:

- 'Viewed Product'
- 'Viewed Category'
- 'Searched Site'
- 'Added to Cart'
- 'Started Checkout'
- 'Order Confirmation'

A Klaviyo Dashboard was checked as well during the above events in both SiteGen & SFRA (Note: data compared using the 'old' / original Klaviyo dashboard view that was active throughout this project).


## Pre-Submission Checklist:
[ x ] This update successfully builds
[ x ] No console errors are displayed on the screen with these updates
[ x ] Console Logs & debugging created during development for debugging that are no longer needed have been removed
[ x ] A description of this update & references to the manual tests has been included in this PR
